Makefile

```diff
@@ -9,7 +9,7 @@ lint:
 hca/api_spec.json:
 	curl https://hca-dss.czi.technology/v1/swagger.json > hca/api_spec.json
 
-test: lint bindings
+test: lint install
```

Comment:
Tests now rely on calling `hca get-files` cmd in a subprocess, so that command has to be installed. Thoughts on this pattern overall?